test/src/e2e_vm_tests/test_programs/ec_recover_test/Forc.toml

```diff
@@ -6,4 +6,4 @@ entry = "main.sw"
 
 [dependencies]
 core = { git = "http://github.com/FuelLabs/sway-lib-core", version = "v0.0.1" }
-std = { git = "http://github.com/FuelLabs/sway-lib-std", version = "v0.0.1" }
+std = { git = "http://github.com/FuelLabs/sway-lib-std", version = "v0.0.2" }
```

Comment:
This will need to pin whichever version has the new B512 code.

Reply:
sounds good. I will see about tagging and releasing a new version of `std`  today.

Reply:
`v0.1.0` of the stdlib has been released if you want to pin this.